transform/mattermost-analytics/models/marts/nps/dim_nps_feedback.sql

```diff
@@ -0,0 +1,23 @@
+select distinct nf.server_id as server_id
+       , nf.user_id as user_id
+       , nf.license_id as license_id
+       , nf.feedback as feedback
+       , ns.score as score
+       , nf.server_version as server_version
+       , COALESCE(cc.license_name, sh.license_name, nf.hosting_type) as hosting_type
+       , nf.user_role as user_role
+       , COALESCE(cc.company_name, sh.company_name) as company_name
+       , COALESCE(cc.customer_email, sh.customer_email) as customer_email
+       , nf.event_date feedback_date
+       , ns.event_date score_date
```

Comment:
3/5 date shouldn't be enough for identifying which records have been sent successfully. There's two solutions for that:
- Use some timestamp or counter that's monotonically increasing (kind of tricky since we're joining feedback date and score dater).
- Use a list of row ids and keep a track of the row ids that have already been updated.

Reply:
Added timestamp and converted to incremental model

Reply:
Removed incremental strategy. Now only has timestamp as a counter to track new data.